feelslikemonday/app/src/main/java/com/example/feelslikemonday/ui/followrequest/FollowerRequestViewModel.java

```diff
@@ -0,0 +1,19 @@
+package com.example.feelslikemonday.ui.followrequest;
+
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.MutableLiveData;
+import androidx.lifecycle.ViewModel;
+
+public class FollowerRequestViewModel extends ViewModel {
+
+    private MutableLiveData<String> mText;
+
+    public FollowerRequestViewModel() {
+        mText = new MutableLiveData<>();
```

Comment:
Out of curiosity, what does MutableLiveData do? What's the difference between it and setting the text attribute of a textview?

Reply:
It is better to ask Ning. I used his format for other fragments.

Reply:
@Starscream29